### PR TITLE
Added release_series function and its test

### PR DESCRIPTION
**Pull request by Wangbarros**
Issue #38

### CODE REVIEW

File: pypixplore/remote.py [hunk 1]
```diff
@@ -51,15 +51,23 @@ def get_latest_releases(self, package_name):
 
     def get_popularity(self, package_name):
         """
-
```
Comment (fccoelho) [inline pypixplore/remote.py]: Please add a description in this docstring

File: pypixplore/remote.py [hunk 2]
```diff
@@ -51,15 +51,23 @@ def get_latest_releases(self, package_name):
...
         :param package_name: name of the package
         :return: dictionary of number of downloads. keys are 'last_month', 'last_week' and 'last_day'
         """
 
         return self._get_JSON(package_name)["info"]["downloads"]
 
     def release_series(self, package_name):
-        raise NotImplementedError
+        """
+            :param package_name: name of the package
```
Comment (fccoelho) [inline pypixplore/remote.py]: MIssing function description

File: pypixplore/remote.py [hunk 3]
```diff
@@ -51,15 +51,23 @@ def get_latest_releases(self, package_name):
...
+            :return: List of itens of the last 10 most recent releases of the package
+        """
+
+        releases_list = list(self._get_JSON("numpy")['releases'].keys())
```
Comment (fccoelho) [inline pypixplore/remote.py]: package name hardcoded

**Comment by Wangbarros** [comment]
Fixed

**Pull request merged**